docs/src/content/docs/guides/i18n.mdx

```diff
@@ -152,14 +152,14 @@ You can provide translations for additional languages you support — or overrid
 
 1. Configure the `i18n` data collection in `src/content/config.ts` if it isn’t configured already:
 
-   ```js
+   ```diff lang="js" ins=", i18nSchema"
```

Comment:
I'm torn about whether to include the comma here. Technically, it's correct, but visually, it seems a bit odd to me 😅. What do you all think?

```suggestion
   ```diff lang="js" ins="i18nSchema"
```